connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -140,6 +140,9 @@ message Read {
 
     // (Optional) A list of path for file-system backed data sources.
     repeated string paths = 4;
+
+    // (Optional) Condition in the where clause for each partition.
```

Comment:
Please add the comment that this currently only works for jdbc.

Reply:
OK